examples/make_eeg_bids.py

```diff
@@ -0,0 +1,173 @@
+Use MNE-Bids for EEG Data
+=========================
+
+In this example, we use MNE-Bids to create a BIDS-compatible directory of EEG
+data. Specifically, we will follow these steps:
+
+1. Download some EEG data from the Open Science Framework (https://osf.io)
+2. Load the data, extract information, and save in a new BIDS directory
+3. Check the result and compare it with the standard
+
+"""
+
+# Authors: Stefan Appelhoff <stefan.appelhoff@mailbox.org>
+# License: BSD (3-clause)
+
+###############################################################################
+# We are importing everything we need for this example:
+import os
+
+from mne import find_events
+from mne.io import read_raw_brainvision
+
+from mne_bids import raw_to_bids
+from mne_bids.datasets import download_matchingpennies_subj
+from mne_bids.utils import print_dir_tree
+
+###############################################################################
+# Step 1: Download the data
+# -------------------------
+#
+# First, we need some data to work with. We will use the example data that is
+# provided with the report on the BIDS extension for EEF data: The "Matching
+# Pennies" dataset. See here for more information: https://osf.io/cj2dr/
+#
+# For convenience, MNE-Bids has a function to download this data.
+
+# make a directory to save the data to
+home = os.path.expanduser('~')
+data_dir = os.path.join(home, 'mne_data', 'eeg_matchingpennies')
+if not os.path.exists(data_dir):
+    os.makedirs(data_dir)
+
+###############################################################################
+# Now, we download data for participant sub-05 (about 350MB). We could also
+# download the whole dataset by specifying `url_dict` as an input to
+# `download_matchingpennies_subj`, see the function's docstring for more
+# information.
+download_matchingpennies_subj(directory=data_dir)
+
+###############################################################################
+# Let's see whether the data has been downloaded using a quick visualization
+# of the directory tree.
+print_dir_tree(data_dir)
+
+###############################################################################
+# The data are part of the example suite for the EEG specification of BIDS, so
+# they are already in BIDS format as we can see from the file names (also, see
+# the `channels.tsv` and `events.tsv` files). However, we want to start from
+# scratch and will only use the "pure" EEG data in its BrainVision format with
+# the following files:
+#
+# * '.eeg' for binary data
+# * '.vhdr' for header information
+# * '.vmrk' for specifying the event markers within the data.
+
+###############################################################################
+# Step 2: Formatting as BIDS
+# --------------------------
+#
+# We are reading the data using MNE-Python's io module and the
+# `read_raw_brainvision` function, which takes a `.vhdr` file as an input.
+vhdr_path = os.path.join(data_dir, 'sub-05_task-matchingpennies_eeg.vhdr')
```

Comment:
actually thinking a bit more, why not add an example for the `physionet` data: https://martinos.org/mne/dev/generated/mne.datasets.eegbci.load_data.html ? The fetcher already exists and there are many subjects (we could just fetch 2 or 3) -- so we can demonstrate how to do the conversion for more than n=1 subjects.

Reply:
This is an excellent idea, I'm on it!